src/FluentCMS.Web.UI/static/scss/components/_button.scss

```diff
@@ -96,6 +96,8 @@ $prefix-button: $prefix + "button";
 
         &:hover {
             @apply text-#{$value}-800;
+			text-shadow: 0 0 2px;
```

Comment:
Couldn’t we use Tailwind classes to do that?

Reply:
I think if it’s not part of the available options in Tailwind, we should drop it.

Reply:
Now I removed this and fixed problem without using text-shadow.